common/lib/dependabot/pull_request_creator/labeler.rb

```diff
@@ -105,7 +105,9 @@ def precision
           new_version_parts = version(dep).split(/[.+]/)
           old_version_parts = previous_version(dep)&.split(/[.+]/) || []
           all_parts = new_version_parts.first(3) + old_version_parts.first(3)
+          # rubocop:disable Performance/RedundantEqualityComparisonBlock
           next 0 unless all_parts.all? { |part| part.to_i.to_s == part }
+          # rubocop:enable Performance/RedundantEqualityComparisonBlock
```

Comment:
This sounds like an upstream false positive, right? I can report it if so.

Reply:
Yes, that's a false positive. It suggests an incorrect fix of `all_parts.all?(part.to_i.to_s)`.